project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject clanhr/shared-deps "0.2.6"
+(defproject clanhr/shared-deps "0.2.6.1"
```

Comment:
I want to leave the first 3 from the original lib, só our changes can be on a 4th level. What do you think?